sdk/python/Makefile

```diff
@@ -1,7 +1,7 @@
 PROJECT_NAME     := Pulumi Python SDK
 LANGHOST_PKG     := github.com/pulumi/pulumi/sdk/v3/python/cmd/pulumi-language-python
-VERSION          := $(shell cd ../../ && pulumictl get version)
-PYPI_VERSION 	 := $(shell cd ../../ && pulumictl get version --language python)
+VERSION          ?= $(shell cd ../../ && pulumictl get version)
```

Comment:
Ah this used to be supported in the get-version script (https://github.com/pulumi/pulumi/blob/master/scripts/get-version#L14) but wasn't translated over to pulumictl. I think the right fix here is that https://github.com/pulumi/pulumictl/blob/master/cmd/pulumictl/version/cli.go should be updated to check for the PULUMI_VERSION envvar.

Reply:
I dont know go so I would rather not patch pulumictl. I am in no hurry so I can wait for an external patch. Meanwhile is it that bad to allow the override here ? Thanks for reviewing this btw.

Reply:
I think we'd want to write this such that PYPI_VERSION (and likewise for the other languages) will either use the expected repo version, or the same string as VERSION. And then there's the question of if we want to call the envvar VERSION or if we should have what get-version script has PULUMI_VERSION.

Reply:
Given the only thing the --language option is doing in pulumictl is handling some edge cases of pypi and pre-release versions I think it might be ok to just have the override login here in the make files.

So VERSION should be set to PULUMI_VERSION if defined, else to shelling out to pulumictl. Likewise for things like PYPI_VERSION and DOTNET_VERSION in the other makefiles.